.gitignore

```diff
@@ -35,3 +35,4 @@
 yarn-debug.log*
 .yarn-integrity
 .env
+.tool-versions
```

Comment:
```suggestion
```

I think you shouldn't ignore this file. Every dev who uses `asdf` will use it so it is important to keep it.